druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/RocksDBLookupExtractor.java

```diff
@@ -47,21 +45,21 @@
     private final RocksDBExtractionNamespace extractionNamespace;
     private RocksDBManager rocksDBManager;
     private LookupService lookupService;
-    private ProtobufSchemaFactory protobufSchemaFactory;
+    private BaseSchemaFactory schemaFactory;
     private KafkaManager kafkaManager;
     private ServiceEmitter serviceEmitter;
     private Cache<String, byte[]> missingLookupCache;
     private final byte[] extractionNamespaceAsByteArray;
 
     public RocksDBLookupExtractor(RocksDBExtractionNamespace extractionNamespace, Map<String, U> map,
                                   LookupService lookupService, RocksDBManager rocksDBManager, KafkaManager kafkaManager,
-                                  ProtobufSchemaFactory protobufSchemaFactory, ServiceEmitter serviceEmitter) {
+                                  BaseSchemaFactory schemaFactory, ServiceEmitter serviceEmitter) {
```

Comment:
how is this going to be instantiated? during query time this factory is used for deserialization. It is instantiated here https://github.com/yahoo/maha/blob/master/druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/cache/MahaNamespaceExtractionCacheManager.java#L391
where concrete class is injected https://github.com/yahoo/maha/blob/master/druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/cache/MahaNamespaceExtractionCacheManager.java#L73

Reply:
fixed,  added the comment in the class MahaNamespaceExtractionCacheManager